### DISCUSSION TITLE
Raspberry Pi 3 with Raspberry pi OS and SDR-RTL.com v3 and SDRPlusPlus

### DISCUSSION THREAD

**Discussion by joanpao** [AlexandreRouma/SDRPlusPlus#1197] [Q&A]
Hi,

I have a Raspberry Pi 3 with Raspberry pi OS and a SDR-RTL.com v3

Now i tested some procedure for install SDR++ but all fail.

Please, I want a fine and actual procedure for install because each procedure fails in one place, maybe a official proceure?

Thanks in avance

**Comment by AndiThurnheer** [comment]
Hi 
First a nice application for the Rapsberry. I have a Pi4b running 64bit OS  The SDR++ seems to start (hour-glas) but never shows the Welcome screen. The same as above a  bit more detailed  instruction in the SDR++ manual to avoid the Debian install pitfalls would help tremendously.
